### PR TITLE
OCM-5260: Add docs about default/worker machine pool

### PR DESCRIPTION
**Pull request by nirarg**

### CODE REVIEW

**Comment by bardielle** [comment]
/lgtm
/approve

please cherry-pick to the v1.5.0-branch as well

File: docs/guides/worker-machine-pool.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+page_title: "Default Machine Pool in Rosa Cluster"
+subcategory: "Description on default machine pool, created as part of rosa cluster"
```
Comment (amalykhi) [inline docs/guides/worker-machine-pool.md:3]:
```suggestion
subcategory: "Description of default machine pool, created as part of Rosa cluster"
```

**Comment by amalykhi** [comment]
except small comment
/lgtm

**Pull request merged**